sql/src/main/java/io/crate/planner/Plan.java

```diff
@@ -34,6 +34,20 @@
  */
 public interface Plan {
 
+    enum StatementType {
+        INSERT,
+        SELECT,
+        UPDATE,
+        DELETE,
+        COPY,
+        DDL,
+        MANAGEMENT,
+        ALL,
```

Comment:
`ALL` seems strange. How can we have an aggregate of other statement types here? Can we add an explanation here via javadoc?

Reply:
It's used here https://github.com/crate/crate/pull/7366/files#diff-6730363f3e03a6b35d6ca1152f239226R121 
Will add a comment why `ALL` is necessary.